static/app/components/smartSearchBar/searchDropdown.tsx

```diff
@@ -289,3 +298,13 @@ const HotkeyGlyphWrapper = styled('span')`
 const HotkeyTitle = styled(`span`)`
   font-size: ${p => p.theme.fontSizeSmall};
 `;
+
+const Invalid = styled(`span`)`
+  font-size: ${p => p.theme.fontSizeSmall};
+  font-family: ${p => p.theme.text.family};
+  color: ${p => p.theme.gray400};
```

Comment:
does it look good on the dark mode too? maybe we should use an alias from our theme?